resources/Linux/raspberrypi/IoTHub/devcontainer.json

```diff
@@ -0,0 +1,10 @@
+{
+  "image": "rpi_app",
+  "name": "rpi_app_container",
+  "dockerFile": "Dockerfile",
+  "extensions": [
+    "ms-vscode.cpptools",
+    "twxs.cmake",
```

Comment:
Why is this extension needed?

Reply:
Because we have CMakeLists.txt as one of our sample files. This extension is for intelliSense of CMake file.